service/src/test/java/uk/gov/hmcts/reform/fpl/service/PopulateCaseServiceTest.java

```diff
@@ -0,0 +1,130 @@
+package uk.gov.hmcts.reform.fpl.service;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mock;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.autoconfigure.jackson.JacksonAutoConfiguration;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+import uk.gov.hmcts.reform.document.domain.Document;
+import uk.gov.hmcts.reform.fpl.enums.State;
+import uk.gov.hmcts.reform.fpl.model.common.DocumentReference;
+import uk.gov.hmcts.reform.fpl.service.time.Time;
+import uk.gov.hmcts.reform.fpl.utils.FixedTimeConfiguration;
+
+import java.util.Map;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.mockito.BDDMockito.given;
+import static org.mockito.Mockito.verify;
+import static uk.gov.hmcts.reform.fpl.utils.DocumentManagementStoreLoader.document;
+
+@ExtendWith(SpringExtension.class)
+@ContextConfiguration(classes = {FixedTimeConfiguration.class, JacksonAutoConfiguration.class})
+class PopulateCaseServiceTest {
+
+    private static final Document MOCK_DOCUMENT = document();
+    private static final Document MOCK_SDO_DOCUMENT = document();
+    private static final Document MOCK_APPLICATION_DOCUMENT = document();
+
+    @Autowired
+    private ObjectMapper mapper;
+
+    @Autowired
+    private Time time;
+
+    @Mock
+    private UploadDocumentService uploadDocumentService;
+
+    private PopulateCaseService service;
```

Comment:
Could it just be autowire + injectmocks? instead of creating a new instance in before each?

Reply:
Worked when changed Mock to MockBean